test/mpi/test_mpi.cc

```diff
@@ -447,57 +458,61 @@ void set_gpu_device(test_set_gpu_device_t set_device)
 #endif
 
 std::vector<ucc_status_t> UccTestMpi::exec_tests(
-        std::vector<std::shared_ptr<TestCase>> tcs, bool triggered)
+        std::vector<std::shared_ptr<TestCase>> tcs, bool triggered,
+                                                    int  persistent)
```

Comment:
why "int persistent"? should be ucc_test_mpi_persistent_t

Reply:
or event just make it "bool" as triggered. and same for inplace

Reply:
changed both inplace and persistent to bool